bitnami/solr/values.yaml

```diff
@@ -710,6 +710,11 @@ ingress:
   ##
   tls: false
 
+
+  ## The name of an existing Secret in the same namespase to use on the generated Ingress resource
+  ## This option will override certManager: true
+  existingSecretName: ~
```

Comment:
Why are we specifying `~`? Shouldn't this be empty?

Reply:
updated